File: BulkyWeb/wwwroot/js/Company.js
$(document).ready(function
    () { loadDataTable(); });

function loadDataTable() {
    dataTable = $('#tblData').DataTable({
        "ajax": { url: '/admin/company/getall' },
        "columns": [
            { data: 'name' },
            { data: 'phoneNumber' },
            { data: 'streetAddress' },
            { data: 'city' },
            { data: 'country' },
            { data: 'zipcode' },
            
            {
                data: 'id',
                "render": function (data) {
                    return `<div class="w-75 btn-group" role="group">
    <a href="/admin/company/upsertCompany?id=${data}" class="btn shadow btn-primary mx-2"><i class="bi bi-pencil-fill"></i>Edit</a>
    <a href="/admin/company/deleteCompany?id=${data}" class="btn shadow btn-danger mx-2"><i class="bi bi-trash3-fill"></i>Delete</a></div>`;
                }
            }]
    });
}

File: Bulky.Models/Models/Company.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bulky.Models.Models
{
    public class Company
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string PhoneNumber { get; set; }= string.Empty;
        public string? StreetAddress { get; set; }
        public string? City { get; set; }
        public string? Country { get; set; }
        public string? Zipcode { get; set; }
    }
}
